Assets/Scripts/UI/StartGameUI.cs

```diff
@@ -0,0 +1,27 @@
+using UnityEngine;
+using UnityEngine.UI;
+using UniRx;
+
+public class StartGameUI : MonoBehaviour
+{
+    [SerializeField] private Button startGameButton;
+    [SerializeField] private Button collectionButton;
+    [SerializeField] private Button tutorialButton;
+    [SerializeField] private Button configButton;
+    [SerializeField] private Button exitButton;
+
+    [SerializeField] private CollectionPage collectionPage;
+
+    //TODO:�U���{��������ӷ|�h���W�h��UIManager������
+    [SerializeField] private UIControlTutorial uIControlTutorial;
+    [SerializeField] private GameObject settingsUI; //TODO:�Ȯɪ����ޥ�GameObject
+
+    private void Start()
+    {
+        startGameButton.onClick.AsObservable().Subscribe(_ => BlueRoseManager.Instance.SceneController.GoToGameScene());
+        tutorialButton.onClick.AsObservable().Subscribe(_ => uIControlTutorial.gameObject.SetActive(true));
+        collectionButton.onClick.AsObservable().Subscribe(_ => collectionPage.gameObject.SetActive(true));
+        configButton.onClick.AsObservable().Subscribe(_ => settingsUI.SetActive(true));
+        exitButton.onClick.AsObservable().Subscribe(_ => BlueRoseManager.Instance.ExitGame());
```

Comment:
這邊應該補上 `AddTo`？這樣在 object 刪掉的時候才會把那些 event listener 移掉
```suggestion
        exitButton.onClick.AsObservable().Subscribe(_ => BlueRoseManager.Instance.ExitGame()).AddTo(this);
```

Reply:
上面幾行也是。

Reply:
![image](https://user-images.githubusercontent.com/75302011/202849649-d496853e-29b6-4927-9695-8f97b59eaa36.png)